.github/workflows/docker-build-push.yml

```diff
@@ -71,15 +60,25 @@ jobs:
 
   test:
```

Comment:
Can you please add an "if" statement on this job to only run on PR events (we don't need to it run on the nightly release for example)?
You can see an example on this same file once it's updated with the version currently on the main branch.